bridge/client/app/_services/data.service.ts

```diff
@@ -37,6 +37,8 @@ export class DataService {
   protected _tracesLastUpdated: { [key: string]: Date } = {};
   protected _rootTracesLastUpdated: { [key: string]: Date } = {};
   protected _projectName: BehaviorSubject<string> = new BehaviorSubject<string>('');
+  protected _uniformDates: {[key: string]: string} = this.apiService.uniformLogDates;
```

Comment:
I think this would then be `{[key: string]: Date}`

Reply:
Because this object is sent to the server, I would have to iterate through all keys to format the values to the right date-string. On read (`localStorage`) I would have to do the same. So I would stay with type `string` here.

Reply:
👍